phpBB/ext/phpbb/teamsecurity/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "phpbb/teamsecurity",
```

Comment:
This is only temp. until we make it an official extension and A51 isn't coming back up without them.